dbt/adapters/glue/impl.py

```diff
@@ -630,21 +630,22 @@ def hudi_merge_table(self, target_relation, request, primary_key, partition_key,
                 'hoodie.bloom.index.update.partition.path': 'true',
             }
 
+        write_mode = 'Append'
         if isTableExists:
-            write_mode = 'Overwrite'
+            # write_mode = 'Overwrite'
```

Comment:
Looks like the logic is swapped.  Should be 
```
if isTableExists:
    write_mode = 'Append'
    ...
else:
    write_mode = 'Overwrite'
    ...
```

Reply:
@armaseg can you help in clarification?